packages/surveys/src/components/general/WelcomeCard.tsx

```diff
@@ -13,6 +13,8 @@ interface WelcomeCardProps {
   timeToFinish?: boolean;
   onSubmit: (data: TResponseData, ttc: TResponseTtc) => void;
   survey: TSurvey;
+  showResponseCount?: boolean;
```

Comment:
Please remove `showResponseCount` and `timeToFinish` as props here and use the survey object instead. You can also set the variables locally in the WelcomeCard component to keep the render conditions the same